deeds-dapp-webapp/src/main/webapp/vue-app/dapp/commonRenting/components/button/RentButtons.vue

```diff
@@ -205,7 +205,7 @@ export default {
     },
     rentalOfferLink() {
       return this.showSeeRentPart
-        && `${this.parentLocation}/marketplace?offer=${this.rentalOffer.id}`;
+        && `${this.parentLocation}/${this.$t('market')}?offer=${this.rentalOffer.id}`;
```

Comment:
Please add using VueX (in main-*.js) a centralized variable to reference the marketplace URL (same for other pages)

BTW, those URLs will have change when changing the language as well using `selectLanguage` method.

```suggestion
        && `${this./marketplaceUrl}?offer=${this.rentalOffer.id}`;
```

Reply:
done!